cmd/kubelet/app/server.go

```diff
@@ -835,15 +835,13 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 			return fmt.Errorf("topology manager policy options %v require feature gates %q enabled",
 				s.TopologyManagerPolicyOptions, features.TopologyManagerPolicyOptions)
 		}
-		if utilfeature.DefaultFeatureGate.Enabled(features.NodeSwap) {
```

Comment:
We don't remove feature gates at this stage.

We technically GA the feature gate and removed n+2 releases later.